R/get_debrief_doc_info.R

```diff
@@ -0,0 +1,20 @@
+#' Get debriefing doc info
+#'
+#' Get the specific information for the upcoming workshop so it can be used in the debriefing doc
+#'
+#' @param ws_dat information about the workshop to be inserted into the debriefing document
+#' @param meta_fld the URL of the meta folder
+#'
+#' @return debrief doc information to be used in rendering the doc
+#'
+#' @export
+#'
+get_debrief_doc_info <- function(ws_dat, meta_fld) {
```

Comment:
`meta_fld` is used as an argument here, but I do not see it show up in the body of the function. Are you sure it is needed?